<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waiterForm</class>
 <widget class="QWidget" name="waiterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meseros</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/img/Images/Waiter.png</normaloff>:/img/Images/Waiter.png</iconset>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
